docs/userGuide/addingPages.md

```diff
@@ -68,7 +68,7 @@ The general approach is as follows:
 1. Create a folder with the path that you want the HTML web page to be available at.
 1. Exclude the files from being further processed by MarkBind by adding them to the <trigger trigger="click" for="modal:addingPages-prevent-processing">`pagesExclude` attribute of the `site.json` file</trigger>.
 1. Generate the HTML files and put them in that folder (This can be done automatically by your build tool, either locally, or as part of your continuous integration process).
-1. Run `markbind build` to generate your MarkBind site.
+1. (Optional) Run `markbind build` to generate your MarkBind site.
```

Comment:
If the user is not using deploy and they deploying it via some other way, they would need to build right?

Reply:
That's right! I seem to have missed that part (and changed the docs too eagerly). 

Thanks for the find! I'll revert the changes for all three affected sections :+1:

Reply:
Fixed in [a27bad9](https://github.com/MarkBind/markbind/pull/2218/commits/a27bad98b61f596abe72f9dfc46d580724ddd2cf)!